### PR TITLE
Create tangorishi.json

### PR DESCRIPTION
**Pull request by tangorishi**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5944"></a>

### CODE REVIEW

**Review by sidharthviz — APPROVED** [review]
LGTM 👍🏼

**Review by iamthecloverly — APPROVED** [review]

File: data/tangorishi.json [hunk 1]
```diff
@@ -0,0 +1,86 @@
+{
+  "name": "Rishi Joshi",
+  "type": "personal",
+  "bio": "Just a guy passionate for Web Development",
+
+  "tags": [ "C++","Open Source", "WebDev" "Javascript", "Java", "C", "Frontend"],
```
Comment (mahfuztbt) [inline data/tangorishi.json:6]: i think accidentally you miss a comma in line (6) between "WebDev" and "Javascript" tag. `"tags": [ "C++","Open Source", "WebDev", "Javascript", "Java", "C", "Frontend"],`

**Review by mahfuztbt — CHANGES_REQUESTED** [review]

File: data/tangorishi.json (hunk 1 above)
Comment (tangorishi) [reply data/tangorishi.json]: yes corrected it.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**